src/library/driftory.ts

```diff
@@ -44,8 +46,8 @@ export default class Driftory {
 
   constructor(args: DriftoryArguments) {
     this.container = args.container;
-    this.onFrameChange = args.onFrameChange;
-    this.onComicLoad = args.onComicLoad;
+    this.onFrameChange = args.onFrameChange || function () { };
+    this.onComicLoad = args.onComicLoad || function () { };
```

Comment:
Since they are optional now I gave them default empty function values to prevent errors.